packages/namekit-react/src/components/DomainOwnershipBadge.tsx

```diff
@@ -11,6 +11,12 @@ import cc from "classcat";
 interface DomainOwnershipBadgeProps {
   domain: DomainCard | null; // when null we show the skeleton
   onClickHandler?: () => void;
+
+  /*
+    The address to compare ownership represents the address
```

Comment:
@FrancoAguzzi so then just rename this field to something intuitive such as `currentUserAddress`.